src/act.drive.cpp

```diff
@@ -821,10 +824,6 @@ ACMD(do_control)
     return;
   }
 
-  if (ch->in_veh) {
```

Comment:
This needs to be restored unless you've tested it exhaustively, it's a safety mechanism to prevent crashes. The code has limited support for actions taken inside vehicles, and riggers have their bodies referenced for about 50% of the actions they take for some reason.

Reply:
Gotcha, worth testing first. I think it makes sense for riggers to be able to work out of vehicles they aren't driving, but I'll set up a test environment and make sure it is functional before changing this part.